homeassistant/components/doorbird/__init__.py

```diff
@@ -270,21 +271,23 @@ def slug(self):
     def _get_event_name(self, event):
         return f"{self.slug}_{event}"
 
-    def _register_event(self, hass_url, event, favs=None):
+    def _register_event(self, hass_url, event, favs=None) -> bool:
```

Comment:
Please type all the arguments when any typing is added.  Thanks!

Reply:
Should be good to go once done.

Reply:
Meh, I knew adding that would come back and bite me. ;) Let me figure out what the others are.

Reply:
Ok to drop the `-> bool:` as well

Reply:
I went ahead and did the functions in the chain.